src/Native/jitinterface/dllexport.h

```diff
@@ -16,25 +16,8 @@
 // ***
 // Define default call conventions
 // ***
-#ifndef _X86_
-
-#define DEFAULT_CALL_CONV
-#define __cdecl
-#define __stdcall
-
-#else // _X86_
-
-#ifndef __stdcall
-#define __stdcall          __attribute__((stdcall))
-#endif
-
-#ifdef PLATFORM_UNIX
-#define DEFAULT_CALL_CONV
+#if _X86_ && !PLATFORM_UNIX
```

Comment:
We use `#if defined...` everywhere else. I cannot think of a reason for why would this not work, but these things always come with surprises.